src/__tests__/useFetch.test.tsx

```diff
@@ -704,25 +694,26 @@ describe('useFetch - BROWSER - errors', (): void => {
     expect(result.current.data).toEqual([])
   })
 
-  it('should reset the error after each call', async (): Promise<void> => {
-    const { result } = renderHook(
-      () => useFetch('https://example.com', { cachePolicy: NO_CACHE }),
+  it('should reset the error after each call', async(): Promise<void> => {
+    const { result, rerender } = renderHook(
+      () => useFetch('https://example.com', { cachePolicy: NO_CACHE })
     )
     expect(result.current.loading).toBe(false)
     await result.current.get()
     expect(result.current.error).toEqual(expectedError)
+    rerender()
```

Comment:
Have had to add this as the second mock request was throwing a DOMException for some reason. Not sure why we're only seeing this issue now. Any ideas?

Reply:
Hm... yeah we should not have to do that. I will take a look first thing in the AM

Reply:
So I have a few things I'd like to go over, but I think the proper way to do this is using `act`

Reply:
the things I'd like to take a closer look at, at some point are how we're adding `error.current` and `data.current` to the `request` object. I also thing we **might** need to remove `loading` from it. It's possible that is what's causing the infinite loops. I'm not 100% sure, just an idea.